cluster.go

```diff
@@ -1687,13 +1691,28 @@ func (c *cluster) considerTopology() error {
 	return nil
 }
 
+// band aid to protect against false nodeLeave events from memberlist
+// the test is the lightest weight endpoint of the node in question /version
+// TODO provide more robust solution to false nodeJoin events
+func confirmNodeDown(uri URI) bool {
+	for i := 0; i < confirmDownRetries; i++ {
+		resp, err := http.Get(uri.Scheme + uri.HostPort() + "/version")
```

Comment:
is there a default timeout on this? we might not want that to be too long, right?

Reply:
I added a timeout of 2 secs but didn't make it configurable